animations/style update

thetylerb opened the pull request:
-Started a new branch based on the master after last nights meeting as to prevent any further diverging branches
-Added a slight fade in effect for the main page (hopefully it helps with loading
-Compressed the background image to improve loading
-Added a field for further page animations in the CSS file
-Now says "search artists" in the search bar once you reach the main page

thetylerb committed:
animations/style update

-Added a fade in effect for the main page (hopefully it helps with loading
-Compressed the background image to improve loading
-Added a field for further page animations
-Now says "search artists" in the search bar once you reach the main page

kcbobbe commented on script.js:
```diff
@@ -268,5 +268,10 @@ $(document).ready(function() {
     }
+//page animations
+    $(".panel").slideDown(1500);
+    $(".columns").fadeIn(7000);
```
On page load, the columns will take 7 seconds to fade in. If the user enters an artist immediately, they have to wait about 6ish seconds for the page to fade in, which is probably too long. If the user waits on the landing page for more than 7 seconds, the animation will not be shown to the user because it already happened in the background (it was just hidden).

kcbobbe commented on script.js:
```diff
@@ -268,5 +268,10 @@ $(document).ready(function() {
       return 0;
     }
+//page animations
+    $(".panel").slideDown(1500);
```
These two animation will not happen again if the user navigates back to the landing page via the navbar brand. They only happen on page load.

thetylerb committed:
favicon upgrade

Added favicon options and changed title of HTML

kcbobbe reviewed (changes requested):
My suggestion would be to get the animations to start not just on page load, but also when certain events happen. For example, the landing page animation should happen when the page loads and when the user navigates back to the landing page. The columns animation should start after the user searches via the landing page.

thetylerb committed:
Favicon update

I added 6 different favicons to have at the top of our page. Feel free to mess with them or get some more at www.logaster.com.
